templates/serverpod_templates/projectname_server_upgrade/Dockerfile

```diff
@@ -1,29 +1,22 @@
-# If you update the dart version, make sure the image is
-# compatible with the busybox image.
 FROM dart:3.2.5 AS build
 
 WORKDIR /app
 COPY . .
 
 RUN dart pub get
-RUN dart compile exe bin/main.dart -o bin/main
+RUN dart compile exe bin/main.dart -o bin/server
```

Comment:
Question: Does this change need to be reflected in any deploy scripts or are we free to change this?

Reply:
no! :)